pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/NamespacesImpl.java

```diff
@@ -241,8 +245,9 @@ public void setNamespaceAntiAffinityGroup(String namespace, String namespaceAnti
     public String getNamespaceAntiAffinityGroup(String namespace) throws PulsarAdminException {
         try {
             NamespaceName ns = NamespaceName.get(namespace);
-            WebTarget path = namespacePath(ns, "antiAffinity");
-            return request(path).get(new GenericType<String>() {});
+            WebTarget path = namespacePath(ns, "policies/AntiAffinityGroup");
```

Comment:
I think that most of the getter methods are not really useful, since there's no advantage from using the general get that returns the entire policies for the namespace. It would be good to remove them from java client / CLI.